src/wallet/rpcwallet.cpp

```diff
@@ -938,9 +938,9 @@ static UniValue getbalance(const JSONRPCRequest& request)
             "\nResult:\n"
             "amount              (numeric) The total amount in " + CURRENCY_UNIT + " received for this account.\n"
             "\nExamples:\n"
-            "\nThe total amount in the wallet with 1 or more confirmations\n"
+            "\nThe total amount in the wallet with 0 or more confirmations\n"
             + HelpExampleCli("getbalance", "") +
-            "\nThe total amount in the wallet at least 6 blocks confirmed\n"
+            "\nThe total amount in the wallet at least 6 blocks confirmation\n"
```

Comment:
```suggestion
            "\nThe total amount in the wallet with at least 6 confirmations\n"
```

Reply:
Hello @PastaPastaPasta,  done the changes